compiler/main/driver.cpp

```diff
@@ -295,6 +295,8 @@ bool fCompilerLibraryParser = false;
 
 chpl::Context* gContext = nullptr;
 
+static bool setChplLLVM = false;
```

Comment:
Minor preference to call this `setChplLLVMEnv` or `compilerOverrodeChplLLVM` or something to indicate that this is not reflecting a user setting, but something the compiler did. (When I read this initially I was assuming it meant the user threw `--llvm`)